scheme/data.json

```diff
@@ -1,3385 +1,1864 @@
 {
   "organizations": {
-    "1": {
+    "f112d652ecf13dacd9c78c11e1e7f987": {
```

Comment:
We probably want this to stay "1" for now right?

Reply:
yeah,